### ISSUE TITLE
Consider swtiching to double-quotes for string fields in instruments

### ISSUE THREAD

**Issue by gdevenyi** [DouglasNeuroInformatics/OpenDataCapture#678]
French uses the single quote `'` fairly regularly in text and it needs to be escaped if single quotes are used for strings.

**Comment by joshunrau** [comment]
Why are you not already able to do that?

**Comment by gdevenyi** [comment]
I should clarify, in the examples.

**Comment by joshunrau** [comment]
We use single quotes everywhere else, which I prefer and am used to. Prettier already accepts double quotes, if the string contains `'`, so I am not sure why this is necessary.

**Comment by joshunrau** [comment]
What about probing the team to see if everyone would prefer the double quotes? If so, then we could change the project formatting settings. I don't think it makes sense to have special rules only for instruments.

**Comment by gdevenyi** [comment]
I ran into some problem when I opened this, but I don't remember what it was anymore. Perhaps I'll catch it again when I build instruments.

For now lets leave this at NEEDSINFO

**Comment by gdevenyi** [comment]
Triaging upstream: this has been open as a style discussion for a long time. Do we want to standardize on double-quotes for instrument string fields, or close this as wontfix and keep current conventions?

**Comment by joshunrau** [comment]
No, I don't want to change this. I am used to single quotes and don't want to update the entire codebase, which would be required if we change the prettier rules.
